sbml/iMA871/exchange.yaml

```diff
@@ -312,6 +312,9 @@ compounds:
   upper: 0.0
 - id: M_PROPALe
   reaction: '1380'
+- id: M_BIOMASS
+  compartment: C-c
```

Comment:
Can we double check the formatting of this and make sure that the compartment designation is correct. Its usually C_c not C-c